packages/trpc/server/routers/viewer/eventTypes.tsx

```diff
@@ -504,7 +504,8 @@ export const eventTypesRouter = router({
       };
     }
 
-    if (input?.price) {
+    if (input?.price || input.metadata?.apps?.stripe?.price) {
+      data.price = input.price || input.metadata?.apps?.stripe?.price;
```

Comment:
Just realised that after v2 we are loosing eventType price attribute from being updated so we read input price or stripe metadata price.